code/game/objects/items/dehy_carp.dm

```diff
@@ -7,6 +7,7 @@
 /obj/item/toy/carpplushie/dehy_carp
 	var/mob/owner = null	//Carp doesn't attack owner, set when using in hand
 	var/owned = 0	//Boolean, no owner to begin with
+	var/mobtype = /mob/living/simple_animal/hostile/carp //So admins can change what mob spawns via var fuckery
```

Comment:
Tbh, should probably make a generic spawn_toy, make the dehydrated carp a subtype of that, and the toy a subtype of dehydrated carp